convert_yml_to_json.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+for file in syntaxes/*.tmLanguage.yaml syntaxes/*.tmLanguage.yml
+do
+  if [ -f "$file" ]; then
+    npx -y js-yaml "$file" > "${file%.*}.json"
```

Comment:
The `-y` flag confirms to `npx` that you want to install `js-yaml` in case it is not currently present